Fix install issues (#94)
QuakeMigrate/QuakeMigrate: docs/installation.rst

- matplotlib < 3.3
- numpy
- obspy >= 1.2
- pandas >= 1 and < 1.1
- pyproj >= 2.6
- scipy

.. note:: These version pins are subject to change. We defer to ObsPy to select suitable versions for NumPy/SciPy.

> .. warning:: Some changes to datetime handling were introduced in matplotlib 3.3, which caused some conflicts with pandas versions <= 1.0.5. It is best to 

Author: ```suggestion
.. warning:: Some changes to datetime handling were introduced in matplotlib 3.3, which caused some conflicts with pandas versions <= 1.0.5. We have pinned these two packages for the time being until we find time to resolve the issue.
```

In addition, we use `NonLinLoc <http://alomax.free.fr/nlloc/>`_ and `scikit fmm <https://pythonhosted.org/scikit-fmm/>`_ as backends for producing 1-D traveltime lookup tables.

NonLinLoc
#########
To download, unpack, and compile NonLinLoc, you can use:

.. code-block:: bash
    
    curl http://alomax.free.fr/nlloc/soft7.00/tar/NLL7.00_src.tgz -o NLL7.00_src.tgz
    tar -xzvf NLL7.00_src.tgz
    cd src

>     make -R all

Author: ```suggestion
make -R all
mkdir bin; export MYBIN=./bin
```

- matplotlib < 3.3
- numpy
- obspy >= 1.2
- pandas >= 1 and < 1.1
- pyproj >= 2.6
- scipy

.. note:: These version pins are subject to change. We defer to ObsPy to select suitable versions for NumPy/SciPy.

.. warning:: Some changes to datetime handling were introduced in matplotlib 3.3, which caused some conflicts with pandas versions <= 1.0.5. A patch was applied, but for the time being we have pinned these two packages until we find time to fully resolve the issues arising from these changes.

In addition, we use `NonLinLoc <http://alomax.free.fr/nlloc/>`_ and `scikit fmm <https://pythonhosted.org/scikit-fmm/>`_ as backends for producing 1-D traveltime lookup tables.

NonLinLoc
#########
To download, unpack, and compile NonLinLoc, you can use:

.. code-block:: bash
    
    curl http://alomax.free.fr/nlloc/soft7.00/tar/NLL7.00_src.tgz -o NLL7.00_src.tgz
    tar -xzvf NLL7.00_src.tgz
    cd src

>     make -R all
>     mkdir bin; export MYBIN=./bin

Author: ```suggestion
    mkdir bin; export MYBIN=./bin
    make -R all
```
